### PR TITLE
Bench and fixes

### PR DESCRIPTION
**Pull request by SonicHDC**
## About the PR

Just steel bench, nothing more. Good for maps and corridors.

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: SonicHDC
- add: Added steel bench!
- fix: Fixed some sprites rotations!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Furniture/chairs.yml [hunk 1]
```diff
@@ -321,3 +321,17 @@
   components:
   - type: Foldable
     folded: true
+
+- type: entity
+  name: steel bench
+  id: SteelBench
+  parent: SeatBase
+  description: A long chair made for metro. Really standart design.
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Furniture/chairs.yml:329]: for **a/the** metro.
standar**d**

**Comment by Flareguy** [comment]
Please don't style these after the old chair sprites. Both the palette and design of them was very ill-fitting for the game as a whole, these probably won't be much different.
Also these need screenshots of the object being used ingame, just an image of the raw sprite won't show how it fits in with the environment.

**Comment by EmoGarbage404** [comment]
In the future, please refrain from creating a PR via uploading files to the github website.

Follow our instructions [here](https://docs.spacestation14.io/en/getting-started/dev-setup) for how to set up git and make a pull request properly.

File: Resources/Prototypes/Entities/Structures/Furniture/chairs.yml [hunk 2]
```diff
@@ -321,3 +321,17 @@
...
     folded: true
+
+- type: entity
+  name: steel bench
+  id: SteelBench
+  parent: SeatBase
+  description: A long chair made for a metro. Really standard design.
+  components:
+  - type: Transform
+    anchored: true
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Furniture/chairs.yml:332]:
```suggestion
    anchored: true
  - type: Physics
    bodyType: Static
```

**Comment by SonicHDC** [comment]
No merge?

**Pull request merged**